src/main/scala/com/gu/i18n/CountryGroup.scala

```diff
@@ -243,7 +243,7 @@ object CountryGroup {
     Country("SN", "Senegal"),
     Country("SO", "Somalia"),
     Country("SR", "Suriname"),
-    //Country("SS", "South Sudan"), Currently not supported by Zuora,
+    Country("SS", "South Sudan"),
```

Comment:
south sudan seems to be supported by zuora according to [the documentation](https://knowledgecenter.zuora.com/BB_Introducing_Z_Business/D_Country%2C_State%2C_and_Province_Codes/A_Country_Names_and_Their_ISO_Codes).

Just in case I tested this locally in subs frontend and I could create a south sudan subscription in UAT without problems